docs/components/power-sensor/_index.md

```diff
@@ -0,0 +1,239 @@
+- `error`: Report any errors that might occur during operation. Successful operation will be `nil`.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/powersensor#PowerSensor).
+
+```Go
+// Create a power sensor instance
+myPowerSensor, err := powersensor.FromRobot(robot, "my_power_sensor")
+if err != nil {
+  logger.Fatalf("cannot get power sensor: %v", err)
+}
+
+// Get the voltage from device in volts
+voltage, isAC, err := myPowerSensor.Voltage(context.Background(), nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetPower
+
+Return the power reading in watts.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra`[(Optional[Dict[str, Any]])](https://docs.python.org/3/library/typing.html#typing.Optional): Pass additional data and configuration options to the [RPC call](/appendix/glossary/#term-grpc).
+- `timeout`[(Optional[float])](https://docs.python.org/3/library/functions.html#float): Specify a time limit (in seconds) for how long `get_power` should wait for a response.
+
+**Returns:**
+
+- `float`: The measurement of the power, represented as a float number.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/power_sensor/power_sensor/index.html#viam.components.power_sensor.power_sensor.PowerSensor.get_power).
+
+```Python
+# Get the power reading from the power sensor
+    power = await my_power_sensor.get_power()
+    print("The power is", power, "Watts")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [Context](https://pkg.go.dev/context): Control the lifecycle of the operation by handling timeouts and managing cancellations.
+- `extra`[(Optional[Dict[str, Any]])](https://docs.python.org/3/library/typing.html#typing.Optional): Pass additional data and configuration options to the [RPC call](/appendix/glossary/#term-grpc).
+
+**Returns:**
+
+- `float64`: The measurement of the power, represented as a 64-bit float number.
+- `error`: Report any errors that might occur during operation. Successful operation will be `nil`.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/powersensor#PowerSensor).
+
+```Go
+// Create a power sensor instance
+myPowerSensor, err := powersensor.FromRobot(robot, "my_power_sensor")
+if err != nil {
+  logger.Fatalf("cannot get power sensor: %v", err)
+}
+
+// Get the power measurement from device in watts
+power, err := myPowerSensor.Power(context.Background(), nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Troubleshooting
+
+You can find additional assistance in the [Troubleshooting section](/appendix/troubleshooting/).
+
+You can also ask questions on the [Viam Community Slack](https://join.slack.com/t/viamrobotics/shared_invite/zt-1f5xf1qk5-TECJc1MIY1MW0d6ZCg~Wnw) and we will be happy to help.
+
+## Next Steps
+
+{{< cards >}}
+  {{% card link="/tutorials/projects/tipsy" %}}
```

Comment:
For Next Steps sections like these, we like to link to tutorials that use the component, to give the user a chance to see the component they just learned about in action. But the Tipsy robot does not use a power sensor I believe. It's fine to not have a next steps section if we don't have any tutorials that use this component.